packages/amplication-server/src/core/workspace/workspace.controller.ts

```diff
@@ -30,4 +30,20 @@ export class WorkspaceController {
     }
     return this.workspaceService.dataMigrateWorkspacesResourcesCustomActions();
   }
+
+  @Post(`createWorkspacesResourcesDefaultCustomActionsMigrationFix/:token`)
+  async createWorkspacesResourcesDefaultCustomActionsMigrationFix(
```

Comment:
Are we missing any kind of authorisation or is it performed in a different level?